packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -45,9 +45,6 @@ function BlockPattern( { isDraggable, pattern, onClick, composite } ) {
 							blocks={ blocks }
 							viewportWidth={ viewportWidth }
 						/>
-						<div className="block-editor-block-patterns-list__item-title">
-							{ pattern.title }
-						</div>
```

Comment:
@mtias Some patterns have descriptions that was already rendered right after the title? Should it be removed as well?

Also, I guess this removal means we need to add a label to the `CompositeItem` component for screen readers.

Reply:
Oh I just saw that there's already a label in its wrapper div, so maybe it's good enough already. We need to test a screen reader.

Reply:
It's a pending design task to explore if a tooltip is helpful too